language/functional_tests/src/config/global.rs

```diff
@@ -70,18 +105,49 @@ pub struct Config {
     /// A map from account names to account data
     pub accounts: BTreeMap<String, AccountData>,
     pub genesis_accounts: BTreeMap<String, Account>,
+    /// The validator set after genesis
+    pub validator_set: ValidatorSet,
 }
 
 impl Config {
     pub fn build(entries: &[Entry]) -> Result<Self> {
         let mut accounts = BTreeMap::new();
+        let mut validator_accounts = 0;
+        for entry in entries {
+            if entry.is_validator() {
+                validator_accounts += 1;
+            }
+        }
```

Comment:
`let mut validator_accounts = entries.filter(|entry| entry.is_validator()).count();`?

Reply:
Thanks! I knew there should be a better Rust-y way to do this..